src/main/scala/com/databricks/labs/overwatch/utils/SparkSessionWrapper.scala

```diff
@@ -20,30 +33,38 @@ trait SparkSessionWrapper extends Serializable {
   @transient
```

Comment:
this @transient should be removed. I know you didn't add it but not sure how it got here. Probably didn't get moved with the spark val like it should have in some earlier PR

Reply:
done. also noticed that _envInit is never being used.